### DISCUSSION TITLE
Data normalizaiton for the dose dependent drug treatment data

### DISCUSSION THREAD

**Discussion by yangyangbian** [vdemichev/DiaNN#1090] [General]
Hi Vadim,
We are treating the same number of cells with different dose of the same drug.
Some doses are higher than its IC50 value.
So, we collect different number of cells after treatment.
We lysate the cell, and measure protein concentration before proteomics sample preparation.
Then we run micro-flow LC-MS based DIA analysis with 5ug of peptides for each dose.
We run the DIA-NN search, and global normalization was enabled.
In the report.pg_matrix file, I summarized number of quantified protein, and median log2 intensity of each dose.

Here is the result.
![image](https://github.com/user-attachments/assets/3e2f719c-24ec-4db4-90c6-01c798de1a1e)

1) the number of quantified protein varied from 2700 to 6800.
2) the median log2 intensity also varied from 17 to 20.

May I ask how to normalize my data?
My project is a bite different, the drug dose treatment induced cell death, and resulting in much lower number of quantified protein at higher dose.
Under this condition, what kind of normalization shall I do before further analysis?
Indeed, I would like to generate a plot of protein intensity vs dose for each protein.
But, before generation of this plot, I need to know how to normalize my data.

By the way, I can shall the raw data if it is needed.

Thank you for your time and I am looking forward to your reply.

**Comment by vdemichev** [comment]
Hi,

Unless you turned off normalisation in DIA-NN, the data in pg_matrix should be normalised and are OK to use as is.

> We lysate the cell, and measure protein concentration before proteomics sample preparation.
Then we run micro-flow LC-MS based DIA analysis with 5ug of peptides for each dose.

If IDs are signficantly different, then it's likely that either (i) some LC-MS problem occurred in some runs or (ii) the injection amount was not adjusted correctly to match the peptide concentration.

You can roughly estimate the total amounts injected by looking at the Total.Quantity column of the .stats.tsv report.

> the median log2 intensity also varied from 17 to 20.

The median of all intensities does not work well if ID numbers are different, so I would filter the data for ubiquitously detected proteins, and median log2 intensity for those will likely be very similar.

> May I ask how to normalize my data?

Most likely it's not necessary to perform further normalisation, but if you wish to perform extra normalisation on the protein level, one simple possibility is to make the median (of hsm https://search.r-project.org/CRAN/refmans/modeest/html/hsm.html) of log-intensities of proteins that are detected in all runs the same. 

Best,
Vadim

**Reply by yangyangbian** [reply]
Hi Vadim,
Thanks so much for your quick reply.
I collect the commonly quantified proteins, and get the median log2 intensity.
Here is the result.
The median log2 intensity decreased at high dose.

**Reply by vdemichev** [reply]
Which likely means that normalisation is OK already. You can plot histograms for different runs (of commonly quantified proteins only) and see if those look like they need any further normalisation.

**Reply by yangyangbian** [reply]
Hi Vadim,
Many thanks to your reply.
I am sorry for the delayed picture. My connection to GitHub failed last time.
Here is the result after I collect the commonly quantified proteins.
![image](https://github.com/user-attachments/assets/1c1e3b14-aac9-467d-8f8e-ff06e72a354c)

**Reply by vdemichev** [reply]
That's quite strange. For all 2794 proteins in 100uM you get 19.85, but for the common subset of 2465, which should be overall more abundant, only 17.86? Seems like some problem with calculation

**Reply by yangyangbian** [reply]
Hey Vadim,
Thanks so much for your reply. 
I am sorry, you are right. There is a mistake.
Here is the right data.
![image](https://github.com/user-attachments/assets/3f609a84-de01-42f0-81ba-d65137c957e3)

**Reply by vdemichev** [reply]
Looks like indeed makes sense to apply some extra normalisation
